draft-ietf-teep-protocol.md

```diff
@@ -1393,22 +1394,24 @@ as discussed in Section 10 of {{I-D.ietf-rats-architecture}}.
 Each freshness mechanism is identified with an integer value, which corresponds to
 an IANA registered freshness mechanism (see the IANA Considerations section of
 {{I-D.ietf-rats-reference-interaction-models}}).
-This document uses the following freshness mechanisms:
+This document uses the following freshness mechanisms which may be added to in the
```

Comment:
Change "added to in the future" to "added in the future"

Reply:
Disagree, as the meaning would be different.  "Added to" means there may be others in the future.  "Added" would mean the ones below are not valid now.

Reply:
You are right; I misread it. Keep it.